demo/dashboard/client/src/components/Spinner.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
It'd be useful to consolidate repeating patterns and components. We're starting to have some duplicates.